src/shared/erros/text-input-email.erros.js

```diff
@@ -0,0 +1,7 @@
+export default class InvalidEmail extends Error {
+    constructor(notEmailValid) {
+        super(notEmailValid);
+        this.message = `${notEmailValid} not a valid e-mail address`;
+        this.name = 'EmailInvalid';
+    }
+}
```

Comment:
End of file must have a blank line

Reply:
![image](https://user-images.githubusercontent.com/62728106/164026860-4cfb54e8-1f77-4947-9a7b-9267319a02a5.png)